wptview.css

```diff
@@ -0,0 +1,73 @@
+:root {
+    font-family: "Fira Sans", "DejaVu Sans", "Bitstream Vera Sans", "Sans";
+    font-size: 12 px;
+    padding: 0;
+    margin:0;
+    background-color: #f8f8f8;
+}
+
+body {
+    margin: 0;
+}
+
+h1 {
+    margin-top:0;
+    box-sizing: border-box;
+    width: 100%;
+    padding: 0.5em;
+    background-color: #e7ebf1;
+}
+
+div#wptview {
+    padding-left: 16px;
+    padding-right: 16px;
+}
+
+table {
```

Comment:
Maybe we could move the table to the right of the import sequence. I find it inconvenient to scroll so much. What do you feel?